web/src/settings_users.ts

```diff
@@ -228,8 +231,23 @@ function populate_users(): void {
 
     if (active_user_ids.length === 0 && deactivated_user_ids.length === 0) {
         failed_listing_users();
+        return;
     }
```

Comment:
@sahil839 is this block a case that can actually happen? I'd expect you can always see yourself here.

Reply:
Yes, there would be atleast one active user, i.e. yourself. So, we can remove this.

Reply:
@sahil839 , should I remove this by making a follow up PR?